main.js

```diff
@@ -1,28 +1,43 @@
 const _ = require('lodash');
 const { app, BrowserWindow, Menu, shell, dialog } = require('electron');
+const prompt = require('electron-prompt');
 
+const { initDB } = require('./src/db/nedb');
+const server = require('./src/index');
 const { Config, setQtumEnv, getQtumExplorerUrl } = require('./src/config/config');
 const logger = require('./src/utils/logger');
 const { blockchainEnv, ipcEvent } = require('./src/constants');
 const Tracking = require('./src/analytics/tracking');
+const Utils = require('./src/utils/utils');
+const Wallet = require('./src/api/wallet');
+
+/*
+* Order of Operations
+* 1. Show select env dialog
+* 2. Start qtumd
```

Comment:
* missing initDB() step before 2
* `initDB()`,`startQtumd`, `start loading window`are executed asynchronously but blocking to kick off following step
* One Todo in future: only local cache transaction is blocking by check wallet encryption, sync blockchain can start once qtumd is up

Reply:
added `set qtum env` & `initDB` steps. and put `start qtumd` & `start loading window` on same line to show asynchronous logic.